srcCpp/RTIDDSImpl.cxx

```diff
@@ -2574,6 +2638,69 @@ unsigned long RTIDDSImpl<T>::GetInitializationSampleCount()
 }
 
 #ifndef RTI_MICRO
+
+
+template <typename T>
+bool RTIDDSImpl<T>::get_serialize_overhead_size(unsigned int &overhead_size)
+{
+    /* Initialize the data elements */
+    T data;
+    data.entity_id = 0;
+    data.seq_num = 0;
+    data.timestamp_sec = 0;
+    data.timestamp_usec = 0;
+    data.latency_ping = 0;
+
+    /* Custom types does not have a bin_data sequence */
+    #ifndef RTI_CUSTOM_TYPE
```

Comment:
should this be RTI_CUSTOM_TYPE or RTI_CUSTOM_TYPE_FLAT_DATA? or the first covers also the second?

Reply:
If `RTI_CUSTOM_TYPE_FLAT_DATA` is enabled, `RTI_CUSTOM_TYPE` will be also enabled. But in any case, if flat data is used, `RTIDDSImpl_FlatData<T>::get_serialized_overhead_size` will be called instead.